scripts/cmd_hubinfo.lua

```diff
@@ -285,7 +288,14 @@ get_tls_mode = function()
     local TLS = ""
     if use_ssl then
         local tls_mode = ssl_params.protocol
-        if tls_mode == "tlsv1" then TLS = "v1.0" else TLS = "v1.2" end
+        if tls_mode == "tlsv1" then 
+            TLS = "v1.0"
+	elseif tls_mode == "tlsv1_2" then
+            TLS = "v1.2" 
+        else
+            TLS = "v1.3"
+        end
+

```

Comment:
Maybe it would be a good idea, to just return ssl_params.protocol. I don't see the benefit of translating that stuff from "tslv1" to "v1.0"...

Also, indentation is again shifted.

Reply:
like that ?
        local tls_mode = ssl_params.protocol
        TLS = tls_mode

Reply:
or 
TLS = string.gsub(tls_mode, "tlsv(%d+)_(%d+)","v%1\.%2")

Although I do not think there will be two or more digit TLS numbers but you may never know :)

Reply:
```
get_tls_mode = function()
  if use_ssl then
    return ssl_params.protocol
  else
    return ""
  end
end
```